src/index.md

```diff
@@ -0,0 +1,18 @@
+# Welcome to Luos documentation
+
+## Introduction 
+
+We started designing Luos with the conviction that building electronic systems should be made easier than it is today. Most of the time should be spent on designing the applications and behaviors instead of on complex and time-and-money-eating technicalities. To give a simple example, adding a new sensor &mdash;for instance a lidar&mdash; to an electronic device in conception should not take more than a few minutes. So you can try, test and iterate fast on a project to truly design what users want.
+
+**Today, Luos is exactly like a <a href="https://en.wikipedia.org/wiki/Microservices" target="_blank">microservices architecture</a>: it encapsulates any software or hardware function to make it communicate and work with any other encapsulated module, however it was developed.**
+
+We have imagined Luos around three key concepts:
+ * **Luos Core:** a tiny library that can be added to the MCU of an embedded board to make it seamlessly communicate with other boards. When conceiving electronic devices, you should be able to compose it from multiple functional parts and not think about communication issues between these parts.
+ * **Luos Driver:** a hardware abstraction to make the integration of a sensor, an effector or a function as easy at it should be &mdash;that is, with one line of code! Drivers must use standardized APIs to let you switch from one type of sensors to another without breaking anything. For example, all types of distance sensors should return a distance in meters using a same function `get_distance()`.
+ * **Robus:** A communication bus dedicated to embedded system communication, to make all the above natural and simple. 
+
```

Comment:
I don't really understand this part. This page is for people discovering what we are doing, they don't care about code parts for now.
Instead we should redirect reader to the relevant part for him by asking some questions and give him the best link to start.
```suggestion
**You are not familiar with Luos operation ?**
 ->  Start by reading the [General use](./_pages/first_steps/general-use.md) page.
 
**You want to make your own board and [Luos drivers](./_pages/modules/drivers.md) or [Luos apps](./_pages/modules/apps.md)?**
 -> Start by reading how to integrate Luos in you [Development environment](./_pages/integrating_luos/dev-env.md).
 -> Then consider to [Create and initialize a module](./_pages/modules/create-init.md).

```